### PR TITLE
Contact mobile form

### PR DESCRIPTION
**Pull request by lawrahkonwea**
This is the contact form mobile portfolio which I created using the;
- Html.
-  Cascading style sheet (CSS).
-  Formspree for data storage.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -232,5 +229,16 @@ <h3 class="lets-connect">LET'S CONNECT</h3>
...
             </div>
         </section>
+    <section class="contact">
+            <h2>Contact Me</h2>
+            <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+        <form action="https://formspree.io/f/xgeqpkgk" method="post">
+            <input type="text" name="text" id="text" placeholder="Name" maxlength="30" required>
+            <input type="email" name="email" id="email" placeholder="Email" required>
+            <textarea name="formMessage" id="message" placeholder="Message" maxlength="500" required></textarea>
+            <button class="form-btn">Get in Touch</button>
```
Comment (ichala) [inline index.html:239]: - [ ] as part of the requirements & to match Figma design kindly add the button interactions such as hover, oressed, etc .. like shown in Figma 

image

File: index.html [hunk 2]
```diff
@@ -232,5 +229,16 @@ <h3 class="lets-connect">LET'S CONNECT</h3>
                 </div>
             </div>
         </section>
+    <section class="contact">
+            <h2>Contact Me</h2>
+            <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+        <form action="https://formspree.io/f/xgeqpkgk" method="post">
+            <input type="text" name="text" id="text" placeholder="Name" maxlength="30" required>
+            <input type="email" name="email" id="email" placeholder="Email" required>
+            <textarea name="formMessage" id="message" placeholder="Message" maxlength="500" required></textarea>
```
Comment (ichala) [inline index.html]: - [ ] to match the Figma design kindly reduce padding for all of your inputs, and try to align the bottom bar in the center

Your Design           | Figma
:-------------------------:|:-------------------------:
image | image

**Review by ichala — CHANGES_REQUESTED** [review]
Hi @lawrahkonwea 
Good job so far! 
👏👏👏 
There are some issues that you still need to work on to go to the next project but you're almost there!

the-simpsons-homer-simpson

## To Highlight 🎉

- Great Design✔️ 
- Validation Works✔️ 
- Formspree used ✔️


 ## Required Changes ♻️

- [ ] 🛑 **_Check the comments below_**

## Optional suggestions

- N/A

**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by cynthiainga — APPROVED** [review]
## STATUS APPROVED ✔️ ✔️ 

Hi @lawrahkonwea, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- [ ] No linters errors ✔️ 
- [ ] Descriptive PR ✔️ 
- [ ] Good commit history ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**